scripts/ci/testing/ci_run_single_airflow_test_in_docker.sh

```diff
@@ -82,6 +82,24 @@ function run_airflow_testing_in_docker() {
     echo
     echo "Semaphore grabbed. Running tests for ${TEST_TYPE}"
     echo
+    local backend_docker_compose=("-f" "${SCRIPTS_CI_DIR}/docker-compose/backend-${BACKEND}.yml")
+    if [[ ${BACKEND} == "mssql" ]]; then
+        local docker_filesystem
+        docker_filesystem=$(stat "-f" "-c" "%T" /var/lib/docker || echo "unknown")
+        if [[ ${docker_filesystem} == "tmpfs" ]]; then
+            # In case of tmpfs backend for docker, mssql fails because TMPFS does not support
+            # O_DIRECT parameter for direct writing to the filesystem
+            # https://github.com/microsoft/mssql-docker/issues/13
+            # so we need to mount an external volume for its db location
+            # the external db must allow for parallel testing so TEST_TYPE is added to the
+            # volume name (see the backend-mssql-bind-volume.yml file)
+            mkdir -p "${HOME}/tmp-mssql-volume-${TEST_TYPE}"
```

Comment:
Should we clean this directory between tries?

Reply:
I even did it initially but i do not think it is needed (we also do not clean the volumes. I think though we s
Could clean them when we clean the volumes (i.e. i. Breeze stop). however this change is only applicable for running tests, so it's not needed.

But that reminded me that I also need to fix breeze execution to include the docker volume and there i will also add the cleanup on stop